enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -971,13 +971,25 @@ func (e *partitionEvictor) computeSize() (int64, int64, int64, error) {
 		})
 	}
 
+	acPrefixRange := func(start, end []byte) ([]byte, []byte) {
+		left := make([]byte, 0)
+		left = append(left, e.acPrefix...)
+		left = append(left, start...)
+
+		right := make([]byte, 0)
+		right = append(right, e.acPrefix...)
+		right = append(right, end...)
+		return left, right
+	}
+
 	// Start scanning the AC.
 	// AC keys look like /partitionID/ac/12312312313(crc-32)/digesthash
-	for i := 0; i < 100; i++ {
-		kr := append(e.acPrefix, []byte(fmt.Sprintf("%.2d", i))...)
-		start, end := keyRange(kr)
-		goScanRange(start, end)
+	for i := 10; i < 99; i++ {
```

Comment:
uber nit: maybe add a comment why it starts @ 10 here, and why 99 is handled as a special case

Reply:
Done.